spec/models/dimensions/edition_spec.rb

```diff
@@ -42,17 +42,6 @@
       expect(results).to match_array(edition1)
     end
 
-    it '.relevant_content' do
-      edition = create :edition, document_type: 'news_story'
-      create :edition, document_type: 'redirect'
-      create :edition, document_type: 'gone'
-      create :edition, document_type: 'vanish'
-      create :edition, document_type: 'unpublishing'
-      create :edition, document_type: 'need'
-      results = subject.relevant_content
-      expect(results).to match_array(edition)
-    end
-
```

Comment:
Does the `relevant_content` method still exist for dimension edition model?

Reply:
Not sure if I get your point here. The `relevant_content` is no longer used so I have removed it.

Reply:
Sorry I missed where you removed them! Makes sense now